openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -685,12 +685,43 @@ def test_add_user_to_cohort(self, mock_tracker):
             lambda: cohorts.add_user_to_cohort(first_cohort, "non_existent_username")
         )
 
+    def test_get_course_cohort_settings(self):
+        """
+        Test that cohorts.get_course_cohort_settings is working as expected.
+        """
+        course = modulestore().get_course(self.toy_course_key)
```

Comment:
this test is verifying the first scenario from the above. Others are being tested in [CourseCohortSettingsHandlerTestCase](https://github.com/edx/edx-platform/pull/7017/files#diff-3044478f11a4053d4c3f34db9c94005fR127)